javascript/webapp/src/hooks/loadProject.ts

```diff
@@ -275,15 +306,39 @@ function preProcessTables(
 	})
 }
 
+// async function processDataTablesOLD(
+// 	pipeline: Pipeline,
+// 	tps?: Workflow[],
+// 	projectFiles?: ProjectFile[],
+// ): Promise<TableContainer | undefined> {
+// 	if (tps !== undefined && tps[0]?.steps?.length && projectFiles?.length) {
+// 		const steps = tps[0]?.steps as Step[]
+// 		pipeline.clear()
+// 		return await runPipelineFromProjectFiles(projectFiles, steps, pipeline)
+// 	}
+// 	return undefined
+// }
+
 async function processDataTables(
-	pipeline: Pipeline,
-	tps?: Specification[],
+	wf?: Workflow[],
 	projectFiles?: ProjectFile[],
+	// setWorkflow?: (workflow: Workflow) => void,
 ): Promise<TableContainer | undefined> {
-	if (tps !== undefined && tps[0]?.steps?.length && projectFiles?.length) {
-		const steps = tps[0]?.steps as Step[]
-		pipeline.clear()
-		return await runPipelineFromProjectFiles(projectFiles, steps, pipeline)
+	if (wf !== undefined && wf[0]?.steps?.length && projectFiles?.length) {
+		const inputs = new Map()
+		projectFiles.forEach(table => {
+			const tableContainer = {
+				id: table.name,
+				table: table?.table as ColumnTable,
+				name: table.name,
+			}
+			inputs.set(table.name, tableContainer)
+		})
+		const [workflow] = wf
+		// setWorkflow && setWorkflow(workflow)
+		const graph = createGraphManager(inputs, workflow)
+		const outputName = [...graph.outputs].pop() || ''
+		return graph.latest(outputName)
```

Comment:
@darthtrevino, Here I tried to replace the `runPipelineFromProjectFiles` with the `createGraphManager` method